docs/News.md

```diff
@@ -1,3 +1,12 @@
+## October 9, 2019
+*  **Added data** consisting of  2615 samples from 3 studies:
+   
```

Comment:
How about https://www.cbioportal.org/study/summary?id=coad_cptac_2019? I am wondering if any other studies were missed?

Please add coad_cptac_2019 to the default set.

Reply:
done